cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+{% import "molecules/social-media.html" as social_media with context %}
+{% set page={"search_description":"Use this tool to double-check that all the details about your loan are correct on your Closing Disclosure. If something looks incorrect or unexpected, ask why."} %}
+{% set active_page = "closing-disclosure" %}
+{% set page_js = "closing-disclosure" %}
+{% block title %}Closing Disclosure Explainer{% endblock title %}
+{% block metadescription %}Use the CFPB"s tool to double-check that all the details about your loan are correct on your Closing Disclosure.{% endblock metadescription %}
+
+{% block banner_top %}
+{% include "_templates/brand-header.html" %}
+{% endblock %}
+
+{% block css %}
+<link rel="stylesheet" href="{{ static('oah/main.css') }}">
+{{ super() }}
+{% endblock css %}
+
+{% block javascript %}
+    <script
+  src="http://code.jquery.com/jquery-3.3.1.slim.min.js"
+  integrity="sha256-3edrmyuQ0w65f8gfBsqowzjJe2iM6n0nKciPUp8y+7E="
+  crossorigin="anonymous"></script>
+    {{ super() }}
+    <script src="{{ static('js/owning-a-home/common.js') }}"></script>
+
+{% endblock javascript %}
+
+{% block content %}
+<main class="content">
+    <div class="content_wrapper">
+        <nav class="breadcrumbs" aria-label="Breadcrumbs">
+            <a href="/owning-a-home/" class="breadcrumbs_link">Owning a Home</a>
```

Comment:
I know the url is currently the same but should the text be "Buying a House"?

Reply:
Fixed.